eng/scripts/Create-APIView.ps1

```diff
@@ -86,6 +86,32 @@ function Get-ResourceProviderFromReadMePath {
     return $null
 }
 
+function Get-ImpactedTypespecProjects {
```

Comment:
Is this logic truly APIView specific? I would feel better if this was in the common functions and shared as I want to avoid different selecting algorithms for each tool that depends on finding typespec projects.

cc @mikeharder

Reply:
OK I do think we might want this to be generalized at some point once we have others that need this functionality.